app/server.py

```diff
@@ -88,10 +95,14 @@ def set_header(r):
 
 
 if __name__ == '__main__':
-    app.run(
-        host='0.0.0.0',
-        port=int(
-            os.environ.get(
-                'PORT',
-                7001)),
-        debug=True)
+
+    if args.dev:
```

Comment:
We can make it look good by not having an if block, maybe use oonditional statement above to define debug.

Reply:
Sorry @AnshulMalik 
Didn't get you. Can you please elaborate more ?

Reply:
Right now you are using if and else block, but both share the same code, what we can do here is, 

```python
app.run(
        host='0.0.0.0',
        port=int(
            os.environ.get(
                'PORT',
                7001)),
        debug=args.dev)
```

Reply:
Awesome @AnshulMalik 
Got it. Missed it though :sweat_smile: